packages/buidler-core/src/internal/solidity/compilationGroup.ts

```diff
@@ -0,0 +1,225 @@
+import { either } from "fp-ts";
```

Comment:
Not sure if we should expose fp-ts in things meant to be used by plugin developers